lib/Conversation/CommentForm/CommentFormInput.js

```diff
@@ -20,14 +23,17 @@ class ContactFormInput extends Component {
   }
 
   render() {
+    const isSubmittedClass = this.props.submitted ? 'is-submitted' : '';
```

Comment:
`isSubmitted` sounds kind of strange;

- is it submitting? (`is-submitting`)
- has it submitted? (`has-submitted`)

Clearing this up would make it easier to understand what it handles 👍